portfolio/src/main/webapp/index.html

```diff
@@ -102,9 +102,16 @@ <h1><span class="text-emphasis">Antonio</span>'s Portfolio</h1>
         </div>
       </div>
       <div id="comments">
+        <form action="/data" method="POST">
+          <p>Enter a comment:</p>
+          <textarea name="text-input"></textarea>
+          <br/><br/>
```

Comment:
nit: Omit the `/` in void elements (https://google.github.io/styleguide/htmlcssguide.html#Document_Type)

You could also probably omit the `<br>`s altogether and get the layout you want using CSS. `display` should help.

Reply:
Changed it to use `display` tags instead of `br`!